q-transform/main.py

```diff
@@ -0,0 +1,48 @@
+    cur_time = datetime.datetime.now()
+
+    #construct the argument parse and parse the arguments
```

Comment:
@hagabbar I'm referring to the entire g-transform/main.py file. It's not a module. It's a script meant to be run by hand, so it should be in the bin directory.